src/main/java/io/reactivex/Observable.java

```diff
@@ -13180,32 +13167,30 @@ public final Completable switchMapCompletableDelayError(@NonNull Function<? supe
     }
 
     /**
-     * Returns an Observable that only emits those items emitted by the source ObservableSource that are not followed
-     * by another emitted item within a specified time window.
+     * Alias to {@link #debounce(long, TimeUnit, Scheduler)}
```

Comment:
Same situation, please keep the first sentence and mention it being an alias later.

Reply:
done